applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py

```diff
@@ -229,44 +229,46 @@ def SolveSolutionStep(self):
     def _add_model_part_containers(self):
 
         domain_size = self.settings["domain_size"].GetInt()
+        if domain_size not in [2,3]:
+            err_msg  = "The input \"domain_size\" is wrong!"
```

Comment:
if you use `'The input "domain_size" is wrong!'` then you can directly use `"` rather than `\"`